hibernate-core/src/main/java/org/hibernate/cfg/Configuration.java

```diff
@@ -64,21 +64,31 @@
  * using {@link MetadataBuilder} and {@link StandardServiceRegistryBuilder}
  * under the covers.
  * <p>
+ * An instance of {@code Configuration} may be obtained simply by
```

Comment:
Like earlier, this can be condensed - developers understand that new == instantiation.